examples/benchmark/tls_bench.c

```diff
@@ -243,6 +243,64 @@ static const unsigned char dhg[] =
 #endif /* !NO_WOLFSSL_SERVER */
 #endif /* !NO_DH */
 
+struct group_info {
+    word16 group;
+    const char *name;
+};
+
+static struct group_info groups[] = {
```

Comment:
Please make `const` if possible.

Reply:
It can't be const because during SetupSupportedGroups() we edit the elements of this array to specify that some of them are not supported.